src/Semantics.js

```diff
@@ -451,6 +519,18 @@ Semantics.createSemantics = function(grammar, optSuperSemantics) {
     s.extendOperationOrAttribute.call(s, 'attribute', name, actionDict);
     return proxy;
   };
+  proxy.getOperationNames = function() {
```

Comment:
I was actually thinking that this is the better thing to do (but was pondering too after Alex suggested this). You might want to know which operations/attributes a semantics supports but you will not access them but rather call them through `s(matchResult).[operationOrAttributeName](...)`.